### PR TITLE
[T4A5][F12-A3]Joel Lim Jing

### PR DESCRIPTION
**Pull request by aljorhythm**
Ready for Review

### CODE REVIEW

File: src/seedu/addressbook/data/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * 
+ * Represent an adding or deleting of a tag for a specific person
```
Comment (okkhoy) [inline src/seedu/addressbook/data/Tagging.java:8]: :+1:

**Review by okkhoy — COMMENTED** [review]
good attempt!
you seem to have missed the next part of the exercise where you need to print the tags added/deleted on exit

**Comment by okkhoy** [comment]
@funnieguy 
Some comments added. Please ack & close the PR after reading comments.

**Comment by aljorhythm** [comment]
ack

**Pull request closed without merging**